assets/sass/components/settings/_googlesitekit-settings-notice.scss

```diff
@@ -204,3 +212,31 @@
 .googlesitekit-settings-notice-ua-cutoff-warning__notice {
 	margin: 0 14px 0 0;
 }
+
+.googlesitekit-settings-notice-adsense-top-earning-pages-widget {
+	align-items: center;
+	padding-bottom: 24px;
+	padding-top: 24px;
+
+	.googlesitekit-settings-notice__icon {
+		margin-right: 20px;
+
+		svg {
+			background-color: $c-surfaces-on-surface;
+			border-radius: 50%;
```

Comment:
Per the above comment, we should be able to simplify this to a `color` for the  `$c-surfaces-on-surface` and the border radius wouldn't be needed since we'd be styling a path of the svg which is already round rather than the element's background itself.

Reply:
Removed as we're using a new SVG.